lib/utils/common.py

```diff
@@ -2431,6 +2439,18 @@ def get_system_commands(port=3000):
         ],
     ]
 
+    uid = os.getuid()
+
+    """
+    Some distros and most containers do not have sudo installed by default. If running
+    as root don't use it. 
+    """
+    if uid == 0:
+        for cmd_list in sys_shell_cmds:
+            for idx, cmd in enumerate(cmd_list):
+                if "sudo " in cmd:
+                    cmd_list[idx] = cmd.replace("sudo ", "")
```

Comment:
Are these just here to document the commands used in node.py? Seems like having a single source of truth would be nice. Like a list of _sysCmd objects defined somewhere that can be used by node.py and be rendered as strings for functions like get_system_commands().

Reply:
no